### ISSUE TITLE
jquery 1.x and IE<=8 not work with default config

### ISSUE THREAD

**Issue by niphor** [polyfillpolyfill/polyfill-service#747]
```
<script src="//cdn.polyfill.io/v2/polyfill.js"></script>
<script src="//code.jquery.com/jquery-1.12.4.js"></script>
<script>
    $(function(){
    $('[name=dd]').filter('.rx').show();
  });
</script>
```

I think Symbol and Event Polyfill brokes jQuery

some thing like `for .. in ..` wil iterate some Symbol key and jQuery does not test it
addEventListener,DOMContentLoaded polyfill is broken

finally I excludes some polyfills,and it seems work again

```
excludes=Event,Event.DOMContentLoaded,Set,Map,WeakMap,Symbol,Symbol.hasInstance,Symbol.isConcatSpreadable,Symbol.iterator,Symbol.match,Symbol.replace,Symbol.search,Symbol.species,Symbol.split,Symbol.toPrimitive,Symbol.toStringTag,Symbol.unscopables
```

is there any others i should exclude ?

is it possible to just exclude Symbol , and automatic exclude all Symbol related polyfills?

Thank you !

**Comment by tracker1** [comment]
Have you tried with jQuery 2 or 3, with jquery-compat?

**Comment by niphor** [comment]
@tracker1

no， i havent try jquery 2 or 3
because maybe css or selector does not support <=ie8

i think jquery-compact is just an alias to jquery 1.x

do you mean jQuery Migrate ？
but jQuery Migrate is not for jquery 3.x support old bowsers

thank you

**Comment by triblondon** [comment]
Hi @niphor can you try to include only enough polyfills to generate the problem?  Rather than using `excludes`, try using `features` to only include the features you want.  If you can nail it down to one feature that would help us figure out the problem (and even better if you can trace the code in jQuery which is triggering the problem).

**Comment by niphor** [comment]
@triblondon 
I opened this issue at 30 Jun, that when cdn.polyfill.io was at 3.7.0

I have said `I think Symbol and Event Polyfill brokes jQuery` in this issue...
1. Event Polyfill
   [#L121](https://github.com/Financial-Times/polyfill-service/blob/v3.8.1/polyfills/Event/polyfill.js#L121)
   `eventElement` maybe is not **function**
   
   this is fixed in 3.9.0rc,I think
2. Symbol Polyfill
   jQuery uses `for .. in ..` [https://github.com/jquery/jquery/blob/1.12.4/dist/jquery.js#L2135](https://github.com/jquery/jquery/blob/1.12.4/dist/jquery.js#L2135)
   so `type` will be something like `__\u0001symbol:iterator0.166322325144687141`
   
   [https://github.com/jquery/jquery/blob/1.12.4/dist/jquery.js#L2136](https://github.com/jquery/jquery/blob/1.12.4/dist/jquery.js#L2136)
   so `matchExpr[ type ]` equal `undefinded`,which not have `exec` method

**Comment by triblondon** [comment]
I don't think compatibility with jQuery 1.x is going to make it to the top of our priority list, unfortunately. Of course we always welcome contributions and that includes a contribution that resolves this issue. Glad to hear that the issue is at least partly resolved by 3.9.0, and if you're still experiencing the second part of the issue we'd welcome a PR with a proposed fix.

**Comment by Sash-OK** [comment]
Set 'default-3.5' works fine in ie8 with JQuery 1.12.4.
